src/main/kotlin/io/kotest/datagen/products.kt

```diff
@@ -0,0 +1,52 @@
+package io.kotest.datagen
+
+import kotlin.random.Random
+
+data class Product(
```

Comment:
Perhaps product is a way too generic case to call this class, isn't it?

I don't think this can generalize all possible products

Reply:
No one class can generialize all products, but this a fake data gen, take what's there and compose it with your own stuff. It's meant to be help or a start.